requirements.txt

```diff
@@ -26,12 +29,22 @@ multidict==6.0.5
     # via
     #   aiohttp
     #   yarl
+packaging==24.2
+    # via pytest
+pluggy==1.5.0
+    # via pytest
 py-cord==2.6.1
 pydantic==2.7.1
     # via pydantic-settings
 pydantic-core==2.18.2
     # via pydantic
 pydantic-settings==2.2.1
+pytest==8.3.4
```

Comment:
separate requirements file for testing deps

Reply:
ignore that, this PR is based off the branch that adds tests. This PR is just for adding the workflow